charts/stable/ubooquity/questions.yaml

```diff
@@ -84,7 +92,12 @@ questions:
       type: dict
       attrs:
 # Include{fixedEnv}
-
+        - variable: MAXMEM
+          label: "MAXMEM"
+          description: "Set the maximum memory in MB"
+          schema:
+            type: int
+            default: 512
```

Comment:
You can also set this in values.yaml and make it a TPL with, lets say, 50% (times 0.5) of the resource limit.

Reply:
This is kinda tricky. As the resource can be also a `string` (eg. 500Mi), not only `int`.

Reply:
Fuck, yes.